### ISSUE TITLE
ValueError in the "Charting an apparition of Venus" example script

### ISSUE THREAD

**Issue by OH6BG** [skyfielders/python-skyfield#581]
https://rhodesmill.org/skyfield/example-plots.html#charting-an-apparition-of-venus

Thank you for the script! Just a note that copying that script from the page and running it will throw a ValueError. Is it just me? Python 3.9.3
Skyfield version: 1.38
jplephem version: 2.15
sgp4 version: 2.18

Traceback (most recent call last):
  File "c:\Users\user\Desktop\Venus\venus.py", line 34, in <module>
    m = planetary_magnitude(apparent)
  File "D:\Programs\Python39\lib\site-packages\skyfield\magnitudelib.py", line 66, in planetary_magnitude
    return function(r, delta, ph_ang)
  File "D:\Programs\Python39\lib\site-packages\skyfield\magnitudelib.py", line 82, in _venus_magnitude
    if ph_ang < 163.7:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

**Comment by brandon-rhodes** [comment]
Well, drat. The script takes advantage of an improvement in Skyfield that, alas, hasn't been released yet. I guess I'll need to do the next release earlier than I had expected, to get the documentation and Skyfield back into agreement. In the meantime you could try out the development version with:
```
pip install -U https://github.com/skyfielders/python-skyfield/archive/master.zip
```

**Comment by OH6BG** [comment]
Thank you, Brandon! It works nicely now :)

**Comment by brandon-rhodes** [comment]
I've just released Skyfield 1.39 so the code should work now without installing from GitHub. Enjoy!
